src/postgres/commands/createPostgresServer/steps/PostgresServerSetCredentialsStep.ts

```diff
@@ -18,7 +18,14 @@ export class PostgresServerSetCredentialsStep extends AzureWizardExecuteStep<IPo
 
     public async execute(context: IPostgresServerWizardContext, progress: Progress<{ message?: string; increment?: number }>): Promise<void> {
 
-        const user: string = nonNullProp(context, 'longUserName');
+        let user: string;
+        // Username doesn't contain servername prefix for Postgres Flexible Servers only
+        // https://docs.microsoft.com/en-us/azure/postgresql/howto-configure-sign-in-aad-authentication
```

Comment:
I don't see anything in this docs about flexible server username stuff. Where exactly does it say that? Just trying to figure out if this is a bug or intended behavior

Reply:
Sorry, that is the wrong link. 
Here: https://techcommunity.microsoft.com/t5/azure-database-for-postgresql/what-is-flexible-server-in-azure-database-for-postgresql/ba-p/1741346 
`"...The direct connection also eliminates the need to include @servername suffix in your username on Flexible Server..."`

I couldn't find a better documentation than this one. But the change is visible on the portal as well.